tools/fp20compiler/rc1.0_general.cpp

```diff
@@ -76,34 +83,56 @@ void GeneralCombinersStruct::Invoke()
 void GeneralCombinerStruct::ZeroOut()
 {
         numPortions = 2;
-        numConsts = 0;
+        ccs.numConsts = 0;
 
         portion[0].ZeroOut();
         portion[0].designator = RCP_RGB;
         portion[1].ZeroOut();
         portion[1].designator = RCP_ALPHA;
 }
 
+int ConstColorsStruct::Count(int reg_name)
+{
+    int count = 0;
+    int i;
+    for (i = 0; i < numConsts; i++)
+        if (cc[i].reg.bits.name == reg_name)
+            count++;
+    return count;
+}
 
-void GeneralCombinerStruct::SetUnusedLocalConsts(int numGlobalConsts, ConstColorStruct *globalCCs)
+void ConstColorsStruct::SetUnusedConst(int reg_name, ConstColorsStruct *globalCCS)

```

Comment:
`SetUnusedConst` was only renamed in second commit. Should already be done in first.